sklearn/gaussian_process/_gpr.py

```diff
@@ -266,10 +272,12 @@ def obj_func(theta, eval_gradient=True):
 
         # Precompute quantities required for predictions which are independent
         # of actual query points
+        # Alg. 2.1, page 19, line 2 -> L = cholesky(K + sigma^2 I)
         K = self.kernel_(self.X_train_)
         K[np.diag_indices_from(K)] += self.alpha
         try:
-            self.L_ = cholesky(K, lower=True)  # Line 2
+            self._L_lower_flag = True
```

Comment:
If it's always True, I'm not sure about creating an attribute for it. Maybe keep it as kwargs `lower=True` everywhere, or define a global constant,
```
CHOLESKY_LOWER = True
```
or
```
CHOLESKY_KWARGS = {"lower": True}
```
?

Reply:
True that we could make a constant for the file